documentation/source/_static/css/custom.css

```diff
@@ -2,9 +2,12 @@
 .wy-side-nav-search>div.version{color:#000000;}
 .wy-menu-vertical a:visited{color:#d9d9d9;}
 .wy-menu-vertical p.caption{color:#3185bc;}
-.wy-nav-content {max-width: none;}
+/* .wy-nav-content {max-width: none;} */
 
 /* Fix horizontal scrolling in tables. See https://github.com/readthedocs/sphinx_rtd_theme/issues/117 and
    https://sphinx-argparse.readthedocs.io/en/stable/misc.html#text-wrapping-in-argument-tables */
 .wy-table-responsive {overflow: visible !important;}
 .wy-table-responsive table td, .wy-table-responsive table th  {white-space: normal; !important;}
+
+/* Remove the '!' on a specific admonition to use it for the testimonials page */
+.admonition.admonition p.admonition-title::before{content:""}
```

Comment:
Ahh, looking into the [documentation on RST admonitions](https://docutils.sourceforge.io/docs/ref/rst/directives.html#generic-admonition), it looks like we might be able to supply a custom class to each `.. admonition::` directive. Something like:

```diff
.. admonition:: Christian Büchel, MD | Universitätsklinikum Hamburg-Eppendorf | Hamburg 🇩🇪
+  :class: testimonial

   SCT is an invaluable tool for our investigations of central pain processing
   in the spinal cord using fMRI. In addition, our combined cortico-spinal
   imaging approach greatly benefits from the accurate spatial prepossessing
   capabilities of the SCT.
```

Then reference the class like this

```suggestion

/* Remove the '!' on a specific admonition to use it for the testimonials page */
.admonition.testimonial p.admonition-title::before{content:""}
```

I'll test this out.

Reply:
that would be super elegant

Reply:
It works! 

![Screenshot from 2021-04-23 10-47-45](https://user-images.githubusercontent.com/16181459/115888852-63318d80-a421-11eb-81ab-069677f5b308.png)

(One has `:class: testimonial`, the other doesn't.)

@jcohenadad since I would be adding `:class: testimonial` to every admonition, I just want to check if you have any pending local changes (e.g. reordering the testimonials), since this change might cause conflicts.

Reply:
> @jcohenadad since I would be adding :class: testimonial to every admonition, I just want to check if you have any pending local changes (e.g. reordering the testimonials), since this change might cause conflicts.

no pending changes-- everything is pushed on my end. Thank you!!